static/js/components/tiles/map_tile.tsx

```diff
@@ -430,10 +435,29 @@ function rawToChart(
 
     const metadataMap = rawData.placeStat.facets || {};
     const placeStat = rawData.placeStat.data[statVarSpec.statVar] || {};
+    // Get the list of child places from either the geojson data or the
+    // placeStat data
+    const childPlaces = !_.isEmpty(rawData.geoJson.features)
+      ? rawData.geoJson.features.map((feature) => feature.properties.geoDcid)
+      : Object.keys(placeStat);
+    let geoJson = rawData.geoJson;
+    if (
+      rawData.enclosedPlaceType in MANUAL_GEOJSON_DISTANCES &&
```

Comment:
Do we need to add an entry for `GeoGridPlace_1Deg` to `MANUAL_GEOJSON_DISTANCES` for 1 degree grids to work?

Reply:
it's already there which is why 1 degree grids work on the old map tool